### PR TITLE
[Android] record ipc exception history and weexCoreThread stackTrace …

### PR DESCRIPTION
**Pull request by lucky-chen**
[Android] record ipc exception history and weexCoreThread stackTrace when white screen or js process died/reload

### CODE REVIEW

File: android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java [hunk 1]
```diff
@@ -876,6 +876,10 @@ public int callReportCrashReloadPage(String instanceId, String crashFile) {
         url = instance.getBundleUrl();
         instance.setHasException(true);
       }
+      Map<String,String> extInfo = new HashMap<>(1);
```
Comment (Darin726) [inline android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java]: map 的 size 为啥不直接设置成 2

File: weex_core/Source/third_party/IPC/IPCFutexPageQueue.cpp [hunk 2]
```diff
@@ -112,6 +112,7 @@ void IPCFutexPageQueue::lock(size_t id, bool checkFinish)
                 break;
             }
             struct timespec waitTime = { m_timeoutSec, 0 };
+            LOGE("IPCException IPCFutexPageQueue:: start futex wait");
```
Comment (Darin726) [inline weex_core/Source/third_party/IPC/IPCFutexPageQueue.cpp]: IPC_LOGE

Comment (lucky-chen) [reply weex_core/Source/third_party/IPC/IPCFutexPageQueue.cpp]: fix done

File: android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java (hunk 1 above)
Comment (lucky-chen) [reply android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java]: fix done

**Pull request merged**